Dictionaries/nld_OCRFixReplaceList.xml

```diff
@@ -26,5 +22,7 @@
   <BeginLines />
   <EndLines />
   <WholeLines />
-  <RegularExpressions />
+  <RegularExpressions>
+    <RegEx find="\bln(?=\p{Ll}{2})" replaceWith="In" />
```

Comment:
Example I shown above will match according to what was removed... I know it won't match for `lnacceptabel` cause it's > than 8...